mandatory/2-function-creation.js

```diff
@@ -16,7 +19,11 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num > 100)
+    num = 100;
+  return `${parseFloat(num.toFixed(2))}%`;
```

Comment:
Can you think of a way to avoid converting a number into a string and then back into a number?